File: app/src/main/java/org/jay/launchstarterexample/tasks/InitBuglyTask.java
package org.jay.launchstarterexample.tasks;


import com.tencent.bugly.crashreport.CrashReport;
import org.jay.appstarter.Task;

public class InitBuglyTask extends Task {

    @Override
    public void run() {
        CrashReport.initCrashReport(mContext, "注册时申请的APPID", false);
    }
}

File: app/src/main/java/org/jay/appstarterexample/tasks/InitBuglyTask.kt
package org.jay.appstarterexample.tasks

import com.tencent.bugly.crashreport.CrashReport
import org.jay.appstarter.Task

class InitBuglyTask : Task() {
    override fun run() {
        CrashReport.initCrashReport(mContext, "注册时申请的APPID", false)
    }
}